simulators/ethereum/sync/mkchain.sh

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
```suggestion
#! /bin/sh
```
There seems to be newline here, did you want a shebang there?

Reply:
I prefer my scripts to not force a particular interpreter on the user. You can run this script using bash, dash, ksh, zsh or even fish.

Reply:
🔥